src/main/cpp/t02_shoes.cpp

```diff
@@ -33,5 +33,27 @@
 using namespace std;
 
 int t02_shoes() {
-    
+    int sizel , kolv ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'kolv' is not assigned a value.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4624556)